package.json

```diff
@@ -420,7 +414,7 @@
     "terser-webpack-plugin": "^5.3.7",
     "ts-loader": "^8.0.3",
     "typescript": "^4.9.5",
-    "webpack": "^5.82.0",
+    "webpack": "^5.82.1",
```

Comment:
Please revert this change and the change to the package-lock file as that is not related to your PR.